### PR TITLE
feat: new profile 3d

### PR DESCRIPTION
**Pull request by netanelC**
New 3d profile.
Added: productSource, productStatus.
Removed: nominalResolution, srsOrigin, productionMethod.
Renamed: absoluteAccuracyLEP90 -> absoluteAccuracyLE90, relativeAccuracyLEP90 -> relativeAccuracySE90.

### CODE REVIEW

File: src/models/layerMetadata/layer3DMetadata.ts [hunk 1]
```diff
@@ -1032,7 +980,6 @@ export class Layer3DMetadata implements ILayer3DMetadata, IMetadataCommonModel {
   @fieldConfig({
     category: FieldCategory.GENERAL,
     isManuallyEditable: true,
-    isAutoGenerated: true,
```
Comment (RonitKissis) [inline src/models/layerMetadata/layer3DMetadata.ts:1035]: why was it removes?

File: src/models/layerMetadata/layer3DMetadata.ts [hunk 2]
```diff
@@ -1182,6 +1102,63 @@ export class Layer3DMetadata implements ILayer3DMetadata, IMetadataCommonModel {
...
+    profile: 'mc_3d',
+    xmlElement: 'mc:productStatus',
+    queryableField: 'mc:productStatus',
+    pycswField: 'pycsw:productStatus',
+  })
+  @catalogDB({
+    column: {
+      name: 'product_status',
+      type: 'text',
+      default: 'UnPublished',
```
Comment (RonitKissis) [inline src/models/layerMetadata/layer3DMetadata.ts:1144]: UNPUBLISHED?

File: src/yaml/3dCatalog/3dMetadata.yaml [hunk 3]
```diff
@@ -193,6 +175,14 @@ components:
           type: string
           nullable: true
           description: Area inside the region
+        productStatus:
+          type: string
+          default: 'UnPublished'
```
Comment (RonitKissis) [inline src/yaml/3dCatalog/3dMetadata.yaml:180]: UNPUBLISHED?

File: src/models/layerMetadata/layer3DMetadata.ts (hunk 1 above)
Comment (netanelC) [reply src/models/layerMetadata/layer3DMetadata.ts:1035]: Because producerName is not auto generated

**Review by RonitKissis — APPROVED** [review]

**Review by RonitKissis — APPROVED** [review]
great work!

**Pull request merged**